lib/assets/javascripts/turbograft/remote.coffee

```diff
@@ -12,6 +12,7 @@ class TurboGraft.Remote
     @refreshOnSuccessExcept = @opts.refreshOnSuccessExcept.split(" ") if @opts.refreshOnSuccessExcept
     @refreshOnError         = @opts.refreshOnError.split(" ")         if @opts.refreshOnError
     @refreshOnErrorExcept   = @opts.refreshOnErrorExcept.split(" ")   if @opts.refreshOnErrorExcept
+    @updatePushState        = @opts.updatePushState
```

Comment:
I'm assuming you mean if they ever use the constructor rather than letting an initializer do it. Currently in admin we have `Shopify.Remote` which is often instantiated directly from within a class. Assuming we want to move those to `Turbograft.Remote` then we already have some places that would do this.